recipes/pyturbo/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "pyturbo" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/twiinit/pyturbo/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: 4e777fd460107ad0d1f948e8f230976381c689cc1e6ac2c754670c51698d8615
```

```suggestion
  url: https://github.com/twiinit/pyturbo/archive/refs/tags/v{{ version }}.tar.gz
  sha256: 4e777fd460107ad0d1f948e8f230976381c689cc1e6ac2c754670c51698d8615
```
Not really needs, just to standardise it.